feat(docs, website): multi path - update submission docs and templates with correct fields (#5561)
loculus-project/loculus: docs/src/content/docs/for-users/submit-sequences.md

## Submit sequences via file upload

> Before starting the upload process, ensure your data is correctly formatted. Every sequence must have a unique ID that links it to its metadata entry.

Reviewer: ```suggestion
Before starting the upload process, ensure your data is correctly formatted. Every sequence must have a unique ID that the metadata entry links  to.
```

Loculus expects:

> - Sequence data in [`fasta` format](../../reference/fasta-format) with a unique `fastaId` per sequence.

Reviewer: ```suggestion
- Sequence data in [`fasta` format](../../reference/fasta-format) with a unique ID per sequence. We call this ID the `fastaId` in what follows. The ID is the start of the header up to and excluding the first white space character.
```

- Metadata for each sample with a unique `id`.
  - When uploading through the API, only `tsv` is supported.
  - When uploading through the website, `xlsx` files are also accepted.
  - Each organism has its own metadata template available on the submission page.
  - You can map columns from your file to the expected metadata fields using the **Add column mapping** option.

> By default, Loculus matches metadata and sequences using `id` and `fastaId` (i.e. the sequence with `fastaId=12` will be joined with the metadata entry with `id=12`). For multi-segmented pathogens, you can provide an additional metadata field called `fastaId` containing a space-separated list of fastaIds to link multiple sequences to a single submission.

Reviewer 2: ```suggestion
Loculus matches metadata and sequences using the `id` column in the metadata (i.e. the sequence with fasta header `>seq_12` will be joined with the metadata entry with `id` of `seq_12`). For multi-segmented pathogens, you can provide an additional metadata field called `fastaId` containing a space-separated list of fasta headers to link multiple sequences to a single submission, e.g. `seq_12_A seq_12_B`.
```

![Metadata template.](../../../assets/MetadataTemplate.png)

### Multi-segmented Pathogens

Multi-segmented pathogens must have one unique `id` per **isolate** (i.e. one per pathogen sample containing all segments). Each segment requires its own unique `fastaId`. Metadata is uploaded per isolate, meaning there will be a single metadata row per `id`. This row should include a `fastaId` field listing all segment `fastaId`s, separated by spaces.

> Segment-specific metadata fields (e.g. `length_{segmentName}`) must be provided individually for each segment.

Author: was just working on that - but the template will have to be different now for multi and single case so I didnt manage to get that nice - I can add the start here though and it can be improved apon
